workspace/packages/local/EditView/package.json

```diff
@@ -259,14 +259,11 @@
      */
 
     "js": [{
-        "path": "resources/lib/ace/src/ace.js",
-        "bundle": true
+        "path": "resources/lib/ace/src/ace.js"
     }, {
-        "path": "resources/lib/String/string.min.js",
-        "bundle": true
+        "path": "resources/lib/String/string.min.js"
     }, {
-        "path": "resources/lib/jsbeautify/lib/beautify.js",
-        "bundle": true
+        "path": "resources/lib/jsbeautify/lib/beautify.js"
```

Comment:
Same as above, there's no need for "bundle" true, this was just confusing the microloader